### ISSUE TITLE
test_size.test_invalid fails on x86

### ISSUE THREAD

**Issue by sbraz** [sahib/rmlint#522]
Hi,
I tried to run the test suite on an x86 chroot and one test fails:
```
test_size.test_invalid ... ('--size 0-18446744073709551616',) did not trigger an error exit.
```

I was able to confirm this by running `rmlint --size 0-18446744073709551616` which indeed does not fail as excected.

**Comment by Cebtenzzre** [comment]
This was fixed by 4043c79992d251442681b72260b5425bd00e9b82. It will be included in master soon.
